projects/uart_test/test/tnt0.c

```diff
@@ -0,0 +1,123 @@
+// send and receive functionality on port tnt0
+#include <fcntl.h>
+#include <stdio.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <termios.h>
+
+/* baudrate settings are defined in <asm/termbits.h>, which is
+included by <termios.h> */
+#define BAUDRATE B38400
+/* change this definition for the correct port */
+#define MODEMDEVICE "/dev/tnt0"
+#define _POSIX_SOURCE 1 /* POSIX compliant source */
```

Comment:
I believe this is already defined in out gcc builds for `<signal.h>`? If not just add it as a build flag.